mitodl/mitxonline
# Add availability to programs in models.py

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5299

### Description (What does it do?)
Adds an availability field to programs which will manually be set to either dated or anytime

### How can this be tested?
Update a program's availability manually in the django admin. Verify that this availability is reflected on the API.  

I opted to programmatically set default to available anytime so as not to cause issue when loading since this is a manually managed field.  There are very few programs, this does not change often, and there is really only one that is very important to modify.  I did run this past Peter and this made sense to him as well.
